### PR TITLE
Packet Loss UI

### PR DESCRIPTION
**Pull request by MaxIsJoe**
CL: [Improvement] The game will now report packet loss to players so we can have more information on odd behaviors.

### CODE REVIEW

File: UnityProject/Assets/Scripts/UI/Systems/MainHUD/PacketLossUI.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Linq;
+using Mirror;
+using UnityEngine;
+using UnityEngine.UI;
+using WebSocketSharp;
+using System.Net.NetworkInformation;
+using System.Threading;
+using Ping = System.Net.NetworkInformation.Ping;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Systems/MainHUD/PacketLossUI.cs:9]: will fail code scanning

File: UnityProject/Assets/Scripts/UI/Systems/MainHUD/PacketLossUI.cs [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		private void Awake()
+		{
+			CancellationTokenSource source = new CancellationTokenSource();
+			token = source.Token;
+			ping.PingCompleted += Loss;
+		}
+
+		private void Update()
+		{
+			MeasurePacketLoss();
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Systems/MainHUD/PacketLossUI.cs:36]: surely there must be some inbuilt methods inside of mirror

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Systems/MainHUD/PacketLossUI.cs:36]: There isn't

File: UnityProject/Assets/Scripts/UI/Systems/MainHUD/PacketLossUI.cs (hunk 1 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Systems/MainHUD/PacketLossUI.cs:9]: which one?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/UI/Systems/MainHUD/PacketLossUI.cs:9]: System.Net.NetworkInformation.Ping; isn't part of the white list

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Systems/MainHUD/PacketLossUI.cs:9]: how do i add this to the white list?

**Pull request closed without merging**